src/test/regress/bin/normalize.sed

```diff
@@ -157,21 +154,6 @@ s/Citus.*currently supports/Citus currently supports/g
 s/prepared transaction with identifier .* does not exist/prepared transaction with identifier "citus_x_yyyyyy_zzz_w" does not exist/g
 s/failed to roll back prepared transaction '.*'/failed to roll back prepared transaction 'citus_x_yyyyyy_zzz_w'/g
 
-# Table aliases for partitioned tables in explain outputs might change
```

Comment:
did you get any flaky test outputs after removing this? Probably not, but would be good to keep an eye on this

Reply:
I didn't get any flaky test outputs.